service/controller/clusterapi/v29/resource/ipam/create.go

```diff
@@ -142,50 +156,39 @@ func (r *Resource) getReservedNetworks(ctx context.Context) ([]net.IPNet, error)
 	return reservedSubnets, nil
 }
 
-func (r *Resource) persistAllocatedNetwork(cr cmav1alpha1.Cluster, azs []string) func(ctx context.Context, subnet net.IPNet) error {
-	return func(ctx context.Context, subnet net.IPNet) error {
-		return r.splitAndPersistReservedSubnet(ctx, cr, subnet, azs)
-	}
-}
-
-func (r *Resource) splitAndPersistReservedSubnet(ctx context.Context, cr cmav1alpha1.Cluster, subnet net.IPNet, azs []string) error {
-	r.logger.LogCtx(ctx, "level", "debug", "message", "updating CR status to persist network allocation and chosen availability zones")
-
-	var providerStatus g8sv1alpha1.AWSClusterStatus
-
-	err := json.Unmarshal(cr.Status.ProviderStatus.Raw, &providerStatus)
+func (r *Resource) getSubnetsFromAWSConfigs(ctx context.Context) ([]net.IPNet, error) {
```

Comment:
The diff is shite. Functionality did not change. Only the interface changed so it is consistent across all lookup functions.

Reply:
The diff is indeed total garbage, but luckily split-screen diff makes it somewhat readable :smile: